Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Input/Controllers/OculusHand.cs

```diff
@@ -132,6 +131,7 @@ public void UpdateHandMaterial(Material newHandMaterial)
 
         public void CleanupHand()
         {
+            teleportPointer.Reset();
```

Comment:
Another question about manually managing the teleport pointer now that we're using the "official" one. I don't think any other input source manages this manually. Do you know why we need to do so here?